test/service_plan_test/service_plan_test.go

```diff
@@ -96,6 +96,138 @@ var _ = test.DescribeTestsFor(test.TestCase{
 				})
 			})
 
+			Describe("GET", func() {
+				var k8sPlatform *types.Platform
+				var k8sAgent *httpexpect.Expect
+
+				BeforeEach(func() {
+					k8sPlatformJSON := common.MakePlatform("k8s-platform", "k8s-platform", "kubernetes", "test-platform-k8s")
```

Comment:
Do we need this platform or can we use smWithBasic and the platform from the test context?

Reply:
The filtering applies only for K8S platforms, so we need to register a new one